### PR TITLE
tlp: add page

### PR DESCRIPTION
**Pull request by mikesprague**
Signed-off-by: Michael Sprague <mikesprague@gmail.com>

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform folder (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

**Comment by mikesprague** [comment]
Relevant `tlp` documentation:

- https://linrunner.de/en/tlp/docs/tlp-linux-advanced-power-management.html
- https://github.com/linrunner/TLP

File: pages/linux/tlp.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# tlp
+
+> Advanced Power Management for Linux.
+
+- Apply settings (according to the actual power source):
+
+`sudo tlp start`
+
+- Apply battery settings (_ignoring_ the actual power source):
```
Comment (sbrl) [inline pages/linux/tlp.md]: @agnivade / @waldyrious, is the italics here ok? I haven't come across this before.

File: pages/linux/tlp.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# tlp
+
+> Advanced Power Management for Linux.
```
Comment (sbrl) [inline pages/linux/tlp.md]: Why is `Power Management` capitalised?

File: pages/linux/tlp.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+`sudo tlp bat`
+
+- Apply AC settings (_ignoring_ the actual power source):
+
+`sudo tlp ac`
+
+- Status report with configuration and all active settings:
+
+`sudo tlp-stat`
```
Comment (sbrl) [inline pages/linux/tlp.md]: This doesn't appear be the same command. Perhaps we could move these to a separate page and insert a 2nd description line that says something like this?

```
> See also `tlp-stat`.

File: pages/linux/tlp.md (hunk 1 above)
Comment (agnivade) [reply pages/linux/tlp.md]: I don't see a grammatical need for italics. Better to remove it.

**Pull request merged**